ReactWindows/ReactNative/ReactNative.csproj

```diff
@@ -231,7 +219,9 @@
       <Name>ChakraBridge</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Folder Include="Views\TextInput\" />
+  </ItemGroup>
```

Comment:
I don't think this is needed.  Make sure you delete the Views\TextInput folder if nothing is currently in it.